### PR TITLE
Fix/ads UI app buttons

### PR DESCRIPTION
**Pull request by JemalSmb**
Buttons styling and other QOL fixes

### CODE REVIEW

File: src/components/ads/AdHandling.js [hunk 1]
```diff
@@ -116,8 +116,8 @@ const AdHandler = () => {
 
 
     return (
-        <div className="pt-10">
-           <div className="container pt-4 pb-4 w-11/12 min-h-screen m-auto flex flex-col">
+        <div className={`pt-10 ${selectedAd ? 'hidden-navbar' : ''}`}>
```
Comment (shnartho) [inline src/components/ads/AdHandling.js:119]:
```suggestion
        <div className=pt-10>
```

Comment (JemalSmb) [reply src/components/ads/AdHandling.js:119]: Yeah it is not needed

Comment (shnartho) [reply src/components/ads/AdHandling.js:119]: because does not seem to be necessary here

File: src/components/VR/vr.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const VR = () => {
 
     return (
         <div className='pt-10' style={{ textAlign: 'center', marginTop: '50px' }}>
-            <h1 style={{ fontSize: '48px' }}>VR Games</h1>
+            <h1 style={{ fontSize: '48px' }}>VR</h1>
```
Comment (shnartho) [inline src/components/VR/vr.js:7]:
```suggestion
```

File: src/components/VR/vr.js [hunk 3]
```diff
@@ -4,7 +4,7 @@ const VR = () => {
 
     return (
         <div className='pt-10' style={{ textAlign: 'center', marginTop: '50px' }}>
```
Comment (shnartho) [inline src/components/VR/vr.js:6]:
```suggestion
        <div className='pt-10'>
```

File: src/components/VR/vr.js (hunk 2 above)
Comment (JemalSmb) [reply src/components/VR/vr.js:7]: That was for when we were initially coding the page.

File: src/components/VR/vr.js (hunk 3 above)
Comment (JemalSmb) [reply src/components/VR/vr.js:6]: That styling is for text that was removed, shouldn't be there.

File: src/components/ads/AdHandling.js [hunk 4]
```diff
@@ -116,8 +116,8 @@ const AdHandler = () => {
 
 
     return (
-        <div className="pt-10">
-           <div className="container pt-4 pb-4 w-11/12 min-h-screen m-auto flex flex-col">
+        <div className=pt-10>
```
Comment (shnartho) [inline src/components/ads/AdHandling.js:119]:
```suggestion
        <div className=`pt-10`>
```

Comment (shnartho) [reply src/components/ads/AdHandling.js:119]: sorry, the syntax would be like this 👍

File: src/components/ads/AdHandling.js [hunk 5]
```diff
@@ -116,8 +116,8 @@ const AdHandler = () => {
 
 
     return (
-        <div className="pt-10">
-           <div className="container pt-4 pb-4 w-11/12 min-h-screen m-auto flex flex-col">
+        <div className=`pt-10`>
```
Comment (shnartho) [inline src/components/ads/AdHandling.js:119]:
```suggestion
        <div className="pt-10">
```

Comment (JemalSmb) [reply src/components/ads/AdHandling.js:119]: Yes correct quotations

**Review by shnartho — APPROVED** [review]

**Pull request merged**